include/ppx/metrics.h

```diff
@@ -299,6 +415,12 @@ class Manager final
     // Get Gauge Basic Statistics, only works for type GAUGE
     GaugeBasicStatistics GetGaugeBasicStatistics(MetricID id) const;
 
+    // Get Live Statistics for a LiveMetric, only works for type GAUGE
+    LiveStatistics GetLiveStatistics(MetricID id) const;
```

Comment:
Shall this be renamed to `GetGaugeLiveStatistics` to match the function above?

Reply:
It does not report `MetricGauge`'s live statistics, so I'll leave Gauge out of the name.